### PR TITLE
addon-centered for Angular

### PR DESCRIPTION
**Pull request by klimentru1986**
## What I did

- added addon-centered for Angular
- added example for addon-centered Angular
- updated addon-centered documentation
- updated ADDONS_SUPPORT.md
- updated tests

### CODE REVIEW

**Review by Hypnosphi — CHANGES_REQUESTED** [review]

File: addons/centered/src/index.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import { window } from 'global';
 import ReactCentered from './react';
 import VueCentered from './vue';
+import AngularCentered from './angular';
```
[the remark this answers is not in the document]
Comment (klimentru1986) [reply addons/centered/src/index.js]: done

**Review by igor-dv — APPROVED** [review]
Very cool =)

**Review by Hypnosphi — APPROVED** [review]

File: lib/cli/haste-map-react-native-packager-1-ce06e11dd572473aa7654c0c19f367c5 [hunk 2]
```diff
@@ -0,0 +1 @@
.../node_modules/lodash/fp/extendAllWith.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/dropRight.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/gte.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/includes.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/conforms.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/findKey.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/pluck.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/reverse.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/tap.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/sortedLastIndexBy.js":["",1483223561000,0,[]],"/home/kliment/Projects
```
Comment (Hypnosphi) [inline lib/cli/haste-map-react-native-packager-1-ce06e11dd572473aa7654c0c19f367c5]: Commited accidentally?

Comment (klimentru1986) [reply lib/cli/haste-map-react-native-packager-1-ce06e11dd572473aa7654c0c19f367c5]: Committed after problems witch Continuous Integration. In commit https://github.com/storybooks/storybook/pull/3573/commits/009f6cd8d8cee7216cbf8cb4a8035d11e9b47d45

`lib/cli/haste-map-react-native-packager-1-ce06e11dd572473aa7654c0c19f367c5` updated after run command `yarn test --cli --update`

Comment (Hypnosphi) [reply lib/cli/haste-map-react-native-packager-1-ce06e11dd572473aa7654c0c19f367c5]: I believe this file wasn't generated by the command

Comment (klimentru1986) [reply lib/cli/haste-map-react-native-packager-1-ce06e11dd572473aa7654c0c19f367c5]: how could i create this big file manually?
It`s generated with command.

Comment (Hypnosphi) [reply lib/cli/haste-map-react-native-packager-1-ce06e11dd572473aa7654c0c19f367c5]: Sure, but probably by some other

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.7`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.7)

**Pull request merged**